CONTRIBUTING.md

```diff
@@ -1,181 +1,163 @@
 
-- Find [“good first issues”](https://github.com/thegetty/quire/issues) tagged in our issue tracker for the main thegetty/quire repo
-- Find [“good first issues”](https://github.com/thegetty/quire-docs/issues) tagged in our issue tracker for the thegetty/quire-docs repo
-- Read the Quire guide to Making Your First Contribution — *in progress*
-- Learn more about contributing to open source at [First Timers Only](http://www.firsttimersonly.com/)
+- [Learn About Contributing To Quire](#about-contributing-to-quire)
+- [Identify a Contribution to Make](#identify-a-contribution-to-make)
+- [Start Work on Contribution](#start-work-on-contribution)
+- [Submit Your Contribution](#submit-your-contribution)
+- [Review Important Resources](#important-resources)
 
-## Important Resources
-
-- [Website](https://quire.getty.edu) 
-- [Issue Tracking](https://github.com/thegetty/quire/issues/)
-- [Documentation](https://quire.getty.edu/documentation)
-- [Forum](https://github.com/thegetty/quire/discussions)
+## About Contributing to Quire
 
-**Quire Core Team:**
+Developed by Getty, Quire is a new, multiformat publishing tool available for immediate access and use. Quire is in a limited beta, © J. Paul Getty Trust, and not yet released as open-source software. **For a free license to use Quire, please complete [this form](http://bit.ly/quire-signup).**
 
-Greg Albers ([@geealbers](https://github.com/geealbers)), product manager<br />
-David Newbury ([@workergnome](https://github.com/workergnome)), product manager<br />
-Matthew Hrudka ([@mphstudios](https://github.com/mphstudios)), lead maintainer<br />
-Erin Cecele Dunigan ([@Erin-Cecele](https://github.com/Erin-Cecele)), community manager<br />
+Striving toward becoming a fully open-source project by spring 2022, we encourage contributions from our community members. No matter what level of experience you have, we welcome all contributions, big and small.
 
-Contact us at [quire@getty.edu](mailto:quire@getty.edu)
+There are two Quire repositories hosted on GitHub that can contribute to:
 
-**All are welcome.** As an open-source community, Quire is committed to providing a safe, welcoming, transparent, and inclusive environment for all our community members and those wishing to become involved. Please see our **[Code of Conduct](https://github.com/thegetty/quire/blob/main/CODE_OF_CONDUCT.md)** for more on the expectations and protections for our community members.
+- [**quire**](https://github.com/thegetty/quire):  Core Quire code including command-line interface, default starter content, and default theme.
+- [**quire-docs**](https://github.com/thegetty/quire-docs): Quire website and documentation.
 
-## Quire Code Repositories
+You are currently in the **quire-docs** repository. The Quire website and documentation are actually built using Quire, so if you haven't already, you will need to [install Quire](https://quire.getty.edu/documentation/install-uninstall/) before making any contributions.
```

Comment:
> The Quire website and documentation are actually built using Quire

I'd like to see this called out more. I feel like if a Quire user knows that contributing to the docs just means working on another Quire project, that's really clarifying and maybe encouraging.